scripts/output_env.sh

```diff
@@ -41,81 +45,100 @@ print_version()
         VERSION_STR=`echo "$VERSION_STR" | $FILTER`
     done
 
-    echo "* ${BIN##*/}: $BIN: $VERSION_STR"
+    echo " * ${BIN##*/}$VARIANT: ${BIN} : ${VERSION_STR} "
 }
 
-print_version "uname" "-a" ""
+echo "Platform:\n"
+if `type "lsb_release" > /dev/null 2>&1`; then
+    echo "Linux variant"
+    lsb_release -d -c
+else
+    echo "Unknown Unix variant"
```

Comment:
```suggestion
    echo "Unknown Linux variant"
```
LSB is a Linux thing. It would make more sense to run this chunk of code only if `uname -s` is `Linux`.

Reply:
I've changed to use `uname` before calling `lsb_release`. Use of `lsb_release` was very deliberate for me, as looking in the logs I really wanted to know which version of the bistro the CI was using at a glance, and it's not always obvious.